op.h

```diff
@@ -164,6 +164,10 @@ Deprecated.  Use C<GIMME_V> instead.
                                 /*  On OP_RETURN, module_true is in effect */
                                 /*  On OP_NEXT/OP_LAST/OP_REDO, there is no
                                  *  loop label */
+                                /*  On OP_OPEN, create a temporary file if the
+                                 *  filename argument is &PL_sv_undef */
```

Comment:
I need to remove this. It crept in during the rebase against blead, but has been reverted.